razorpay/resources/base.py

```diff
@@ -1,3 +1,6 @@
+from ..constants.url import URL
```

Comment:
why is this needed?

Reply:
was relevant in the previous commit, forgot to remove this